extension/snippets/dvc-yaml.code-snippets

```diff
@@ -13,123 +13,110 @@
   "Pipeline Stage Template": {
     "prefix": "dvc-pipeline-stage",
     "body": [
-      "  ${1:stage name}:",
-      "    cmd: ${2:command for the stage, e.g. python train.py}",
+      "  ${1:train}:",
+      "    cmd: ${2:python train.py}",
       "    deps:",
-      "      - ${3:list dependencies for the stage, e.g. train.py}",
+      "      - ${3:train.py}",
       "    params:",
-      "      - ${4:list params file(s) for the stage, e.g params.yaml}",
-      "    metrics:",
-      "      - ${5:list metric file(s) for the stage, e.g metrics.json}",
+      "      - ${4:params.yaml}",
       "    outs:",
-      "      - ${6:list output(s) of the stage, e.g. model.pkl}"
+      "      - ${6:model.pkl}"
     ],
     "description": "Stage template for dvc.yaml pipeline"
   },
-  "Minimal Pipeline Stage Template": {
-    "prefix": "dvc-pipeline-minimal-stage",
-    "body": [
-      "  ${1:stage name}:",
-      "    cmd: ${2:command for the stage, e.g. python train.py}",
-      "    deps:",
-      "      - ${3:list dependencies for the stage, e.g. train.py}",
-      "    outs:",
-      "      - ${4:list output(s) of the stage, e.g. model.pkl}"
-    ],
-    "description": "Minimal stage template for dvc.yaml pipeline"
```

Comment:
[F] After removing metrics params where the only difference between this and the snippet above. I didn't think it made sense to keep it around.